docs/enhancements/tenancy-based-auth.md

```diff
@@ -0,0 +1,103 @@
+# Tenancy Based Authorization
+
+## Summary
+The current authorization scheme in the SaaS grants read and write access only to the cluster's owner.
+To align with the OCM authorization scheme, we would like to allow the following:
+- A tenancy-based read-only access to resources. 
```

Comment:
Which environment are we talking about here?

REST vs kube-api? If REST, on-prem or OCM/redhat.com, or both?

Reply:
SaaS - it says it in the first sentence.

Reply:
Ah missed that, thanks